hadoop-ozone/dist/src/main/k8s/examples/minikube/config-configmap.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   name: config
 data:
   OZONE-SITE.XML_hdds.datanode.dir: /data/storage
-  OZONE-SITE.XML_ozone.scm.datanode.id: /data/datanode.id
+  OZONE-SITE.XML_ozone.scm.datanode.id.dir: /data/datanode.id
```

Comment:
Sorry to comment so later, is this wrong ? Shouldn't this be /data instead of /data/datanode.id ?